language/stdlib/modules/LibraAccount.move

```diff
@@ -307,11 +318,33 @@ module LibraAccount {
 
     // Withdraw `amount` Libra<Token> from the account balance under
     // `cap.account_address`
-    public fun withdraw_from<Token>(cap: &WithdrawCapability, amount: u64): Libra<Token>
-    acquires Balance, AccountOperationsCapability {
-        let account_balance = borrow_global_mut<Balance<Token>>(cap.account_address);
-        // The sender has retained her withdraw privileges--proceed.
-        withdraw_from_balance<Token>(cap.account_address, account_balance, amount)
+    fun withdraw_from<Token>(
+        cap: &WithdrawCapability,
+        payee: address,
```

Comment:
This is going to make account limits and making sure we don't count intra-vasp transfers so much easier as well!